src/styles/StudentStyles/index.less

```diff
@@ -144,6 +146,108 @@
     border-radius: 25px;
 }
 
+.student-story-card h2 {
+    font-size: 1.8rem;
+}
+
+.student-story-card h4 {
+    font-size: 1.2rem;
+}
+
 .student-story-card img {
     border-radius: 100%;
-}
\ No newline at end of file
+}
+
+.top-story-card {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.top-left {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 250px;
```

Comment:
I would change this to a more dynamic unit, like %, rem, em, etc., since this seems to be a big chunk of space that might be noticeably different if the screen is smaller.